src/HTMLImports/importer.js

```diff
@@ -128,22 +128,28 @@ function makeDocument(resource, url) {
   var doc = document.implementation.createHTMLDocument(IMPORT_LINK_TYPE);
   // cache the new document's source url
   doc._URL = url;
-  // establish a relative path via <base>
-  var base = doc.createElement('base');
-  base.setAttribute('href', url);
-  // add baseURI support to browsers (IE) that lack it.
-  if (!doc.baseURI && !hasBaseURIAccessor(doc)) {
-    // Use defineProperty since Safari throws an exception when using assignment.
-    Object.defineProperty(doc, 'baseURI', {value:url});
+  
+  // install html
+  doc.body.innerHTML = resource;
```

Comment:
My thinking was so the doc content would be set so we could search for any existing `<base>` tags with the `doc.querySelector('base')` line below.  Is there a concern with this?